README.md

```diff
@@ -2,10 +2,43 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
+The TSC is community and contributor driven and is a home for community managed 
+projects and assets. The mission of the Node.js Foundation is to enable and encourage 
+the adoption of Node.js and the TSC is the home of the community driven portion
```

Comment:
I don't think we need the second half of this sentence, it's telling the TSC to stay out of anything not "community driven", which is vague but also not in the spirit of how the Foundation is set up. Although it may be in the spirit of how the Foundation currently operates, with two strictly divided parts, I don't believe this is optimal or in Node's best interests.

> The mission of the Node.js Foundation is to enable and encourage the adoption of Node.js.

Ought to be enough to guide the TSC.